libs/micronaut-snitch/micronaut-snitch.gradle

```diff
@@ -27,11 +27,16 @@ dependencies {
     implementation "io.micronaut:micronaut-runtime"
     implementation "io.micronaut:micronaut-http-client"
 
-    runtimeOnly "ch.qos.logback:logback-classic:1.2.3"
+    // Micronaut 3 Compatibility
+    annotationProcessor 'com.google.code.findbugs:jsr305:3.0.2'
+    implementation 'com.google.code.findbugs:jsr305:3.0.2'
```

Comment:
micronaut 3 does not us findbugs but its own set of nullability annotations.

Reply:
I know - that's the reason why it's missing on the classpath for MN 3.x builds. but it still understands it, doesn't it?